pom.xml

```diff
@@ -115,6 +115,12 @@
             <version>1.650</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.main</groupId>
+            <artifactId>jenkins-test-harness</artifactId>
+            <version>1.644</version>
```

Comment:
There is no 1.650 version available for the jenkins-test-harness. The 1.644 however is good enough for our tests :-)